operator-metadata/manifests/bundle.clusterserviceversion.yaml

```diff
@@ -823,7 +874,7 @@ spec:
     
     The AMQ Streams Operator understands how to run and upgrade between a set of Kafka versions.
     When specifying a new version in your config, check to make sure you aren't using any features that may have been removed.
-    See [the upgrade guide](https://access.redhat.com/documentation/en-us/red_hat_amq_streams/2.4/html/deploying_and
+    See [the upgrade guide](https://access.redhat.com/documentation/en-us/red_hat_amq_streams/2.5/html/deploying_and
```

Comment:
@PaulRMellor This will be the correct link at release time right? I am getting a 404

Reply:
I just changed the version :P ... hoping that the link will work at some point :-D

Reply:
Yeah. That should be fine when published. If we don't want to keep changing version, we could change to something like:

`For more information on upgrading, see the [documentation for AMQ Streams on OpenShift](https://access.redhat.com/documentation/en-us/red_hat_amq_streams/).`

Reply:
Yeah I like it, I will do this way.